docs/UserGuide.md

```diff
@@ -191,31 +198,31 @@ Format: `label t>TAG_NAME l>LABEL1 [l>LABEL2]...`
 ![label](images/screenshots/label_command.png)
 
 Examples:
-* `label t>file1 l>important` (Add a label of tag name `file1` with label name `important`) 
-* `label t>file2 l>important l>exam` (Add a label of tag name `file2` with multiple labels of label name `important` and `exam`)
+* `label t>file1 l>important` (Add a label `important` to an existing tag name `file1`) 
+* `label t>file2 l>important l>exam` (Add labels of label name `important` and `exam` to an existing tag name `file2`)
 
 ### Deleting multiple labels from a tag : `unlabel`
 
-Delete one or more label from a tag. <br>
+Deletes one or more labels from a tag. <br>
+You can delete an unwanted label from a certain tag in order to uncategorized the tag.
```

Comment:
personally I feel the two sentences are overlapping with each other, since deleting label is mentioned twice

Reply:
I want to give reason on why we would want to use unlabel. But yeah, I agree that it kinda repeating the same sentence. I'll change it!